package.json

```diff
@@ -3,8 +3,8 @@
   "version": "2.20.2",
   "description": "Import with sanity.",
   "engines": {
-    "node": ">=4"
-  },
+    "node": "^10.12.0 || >=12.0.0"
```

Comment:
This shouldn't be here. Whilst [ESLint 7.0.0 requires it](https://eslint.org/docs/user-guide/migrating-to-7.0.0#drop-node-8), eslint-plugin-import shouldn't require Node 10+ as it can still be used with ESLint 6.x.

Reply:
Thanks for the review. Reverting changes to the old node engine requirement - >=4 ( although I think that is fairly old ! )